scripts/deploy.py

```diff
@@ -188,8 +186,11 @@ def upload_archive(self, upload, archive):
                 data=fp,
                 params=self.params,
             )
-            if response.status_code != 200:
-                sys.exit("Trouble uploading artifact %s to bucket" % archive)
+            if response.status_code not in [200, 201]:
+                sys.exit(
+                    "Trouble uploading artifact %s to bucket with response code %s" %
+                    archive, response.status_code
+                )
```

Comment:
Thanks for the merge - as I was debugging I ran through all my actions so couldn't replicate the issues.  Went back to try to replicate all the issues to have screen grabs.

In any case - not terribly good at python - IDK if this is correct string interpolation or if it needs `(archive, response.status_code)` or if `archive, response.status_code` is correct syntax here.

Reply:
Sorry I missed this! I would use format strings now:

```python
sys.exit(f"Trouble uploading artifact {archive} to bucket with response code {response.status_code}")
```